### PR TITLE
Remove unused code and simplify string format.

### PR DESCRIPTION
**Pull request by godaji**
I checked the code and it was pretty clean. I found and fixed some minor improvements. Please reflect if you wish.

### CODE REVIEW

File: gitlab/tests/test_base.py [hunk 1]
```diff
@@ -93,7 +90,6 @@ def test_pickability(self):
         self.assertIsInstance(unpickled, FakeObject)
         self.assertTrue(hasattr(unpickled, "_module"))
         self.assertEqual(unpickled._module, original_obj_module)
-        pickled2 = pickle.dumps(unpickled)
```
Comment (max-wittig) [inline gitlab/tests/test_base.py]: I think this is supposed to be a pickle object test.

**Comment by max-wittig** [comment]
Thanks for the merge request! Could you just please follow our commit conventions and formulate the commit message like so: 

> refactor: remove unused code, simplify string format

**Comment by max-wittig** [comment]
> Thanks for the merge request! Could you just please follow our commit conventions and formulate the commit message like so:
> 
> > refactor: remove unused code, simplify string format

Your commit messages are the reason why the tests are failing. More info: https://github.com/python-gitlab/python-gitlab#contributing

File: gitlab/tests/test_base.py (hunk 1 above)
Comment (godaji) [reply gitlab/tests/test_base.py]: Ok, I will rollback it. :)

**Comment by max-wittig** [comment]
@godaji Thanks for the fixes!

**Pull request merged**